src/test/java/e2e/LoginTest.java

```diff
@@ -0,0 +1,57 @@
+package e2e;
+
+import e2e.pages.LoginPage;
+import e2e.util.DataProviders;
+import org.testng.annotations.Test;
+
+public class LoginTest extends TestBase {
+    LoginPage loginPage;
+
+    @Test(dataProvider = "userCanLogin",dataProviderClass = DataProviders.class)
+    public void userCanLogin(String email, String password){
+       // String email = "standard_user";
+       // String password = "secret_sauce";
+
+        loginPage = new LoginPage(app.driver);
+        loginPage.login(email, password);
+
```

Comment:
need to add check that user was logged 
example : use waitForLoading for Products page - for positive login test
Fo negative login test need to check that user stay on the login page, you can use after login waitForLoading fo Login page

Reply:
I made changes